cou/cli.py

```diff
@@ -291,4 +292,6 @@ def entrypoint() -> None:
     finally:
         if args.command == "upgrade":
             loop.run_until_complete(run_post_upgrade_sanity_check(args))
+        if log_file:
+            progress_indicator.stop_and_persist(text=f"Full execution log: '{log_file}'")
```

Comment:
We should not rely on `progress_indicator` for printing text: it can be disabled with `cou -q`

Reply:
Change to `print`